vars/all.yml

```diff
@@ -10,3 +10,4 @@ manifest_version: release
 docker_tag: latest
 
 ### Components ###
+enable_storage: no
```

Comment:
Shoud [Readme](https://github.com/kubevirt/kubevirt-ansible/blob/master/roles/kubevirt/README.md) mention that var and it defaults to no?